### ISSUE TITLE
Outdated dependencies

### ISSUE THREAD

**Issue by StijnRuts** [waterson/purescript-webaudio#5]
I'm getting version conflicts when trying to install purescript-webaudio trough bower, and build with pulp.

Steps to reproduce:
```
$ pulp init

$ bower install --save purescript-webaudio
bower  ENOTFOUND Package purescript-webaudio not found

$ bower install --save https://github.com/waterson/purescript-webaudio.git#master
Unable to find a suitable version for purescript-eff, please choose one by typing one of the numbers below:
...

$ bower install --force-latest --save https://github.com/waterson/purescript-webaudio.git#master

$ pulp build

Error 1 of 35:

  at bower_components/purescript-nullable/src/Data/Nullable.purs line 19, column 35 - line 19, column 35

    Unable to parse module:
    The `*` symbol is no longer used for the kind of types.
  The new equivalent is the named kind `Type`.


  See https://github.com/purescript/documentation/blob/master/errors/ErrorParsingModule.md for more information,
  or to contribute content related to this error.

Error 2 of 35:
    ...
```

Probably, the dependencies in bower.json should have a version defined other than "*"
https://github.com/waterson/purescript-webaudio/blob/master/bower.json#L18

**Comment by newlandsvalley** [comment]
I think this is resolved in #3

**Comment by StijnRuts** [comment]
#3 does indeed resolve this
